lib/SVG/Graph/Pie.rb

```diff
@@ -219,13 +219,12 @@ def draw_data
           max_value = max_value < x ? x : max_value
           total += x
         }
-        percent_scale = 100.0 / total
 
         prev_percent = 0
         rad_mult = 3.6 * RADIANS
         @config[:fields].each_index { |count|
           value = @data[count].to_f
-          percent = percent_scale * value
+          percent = 100.0 * value / total
           radians = prev_percent * rad_mult
 
           if percent == 100.0
```

Comment:
Actually it causes numerical issues. Checking equality of two float numbers isn't a valid method. We should use some delta range.